### PR TITLE
Fix #3818 : Expand and collapse story when launched from Recently Played

### LINKED ISSUE #3818: Chapter Expand/Collapse not working when launched from Recently-Played
**Describe the bug**
The expand and collapse feature doesn't work correctly when the topic's lesson tab opened from recently played cards from the Home Screen. 

**To Reproduce**
1. Go to the home screen 
2. Play some exploration
3. Go back to the home screen, the recently played exploration will appear
4. Click on any card and lesson tabs open
5. Click on expand collapse icon - it will not work

**Expected behavior**
It should expand and collapse correctly. 

**Device**
- Device being used
- SDK version - Android 11

### PR DESCRIPTION
**Pull request by darkmat13r**
## Explanation

Fixed the collapse and expand functionality when story played from Recently Played from home screen.

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [ ] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

Before: 

https://user-images.githubusercontent.com/35571567/135723747-61f3a4ab-dad0-492d-ac82-c723611021b5.mp4

After: 

https://user-images.githubusercontent.com/35571567/135719290-6b76220e-b5ef-4b02-a9fd-dafc1840f054.mp4

Portrait Mode:

https://user-images.githubusercontent.com/35571567/135723780-84c977f3-24aa-4b85-aaa3-763e9a15afd5.mp4

### CODE REVIEW

**Comment by darkmat13r** [comment]
@rt4914 PTAL

File: app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt [hunk 1]
```diff
@@ -777,6 +777,69 @@ class TopicLessonsFragmentTest {
...
+          recyclerViewId = R.id.story_summary_recycler_view,
+          position = 1,
+          targetViewId = R.id.chapter_recycler_view
+        )
+      ).check(matches(not(isDisplayed())))
+    }
+  }
+
+  @Test
+  fun testLessonsPlayFragment_loadRecentStory_clickExpandIcon_changeConfiguration_chapterListIsNotVisible() { // ktlint-disable max-line-length
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt]: We should avoid using `ktlint-disable`.
You can fix this by writing it like this:
`testLessonsPlay_loadRecentStory_clickExpandIcon_land_chapterListIsNotVisible`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks @darkmat13r . One nit change suggested.

File: app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt (hunk 1 above)
Comment (darkmat13r) [reply app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt]: @rt4914 Done.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**